src/scss/lib/components/navbar.scss

```diff
@@ -0,0 +1,137 @@
+.navbar{
+  background-color: transparent;
+  padding: 22px 0 0;
+  .navbar-item{
+    color: #ffffff;
+    font-family: $font-menu,sans-serif;
+    font-size: 14px;
+    line-height: 18px;
+    background-color: transparent;
+    font-weight: 400;
+    align-self: center;
+    letter-spacing: 0.6px;
+    word-spacing: -2px;
+    margin-left: 41px;
+    &:hover{
+      background-color: transparent;
+      color: $gray-400;
+      transition: 0.3s ease-out;
+    }
+  }
+  .navbar-brand__logo{
+    position: relative;
+    left: 11px;
+    @include mobile{
+      margin-left: 5px;
+    }
+    img{
+      width: 31px;
+      height: 27px;
+    }
+  }
+  .navbar-menu{
+    position: relative;
+    right: 80px;
+    .button{
+      font-size: 11px;
+      font-weight: 400;
+      line-height: normal;
+      height: 32px;
+      width: 108px;
+      margin-left: 54px;
+      &:hover,&:active,&:focus{
+        background-color: transparent;
+        outline: none;
+        box-shadow: none;
+      }
+    }
+  }
+  .navbar-burger{
+    margin-right: 25px;
+    transition: 0.3s ease-out;
+    &.is-active{
+      span:nth-child(3) {
+        width: 34px;
+      }
+      span:nth-child(3) {
+        -webkit-transform: translateY(-5px) rotate(-45deg);
+        transform: translateY(-10px) rotate(-45deg);
+      }
+    }
+    span{
+      background-color: #ffffff;
+      width: 34px;
+      transition: 0.3s ease-out;
+      &:nth-child(2) {
+        top: calc(55% - 1px);
+      }
+      &:nth-child(3) {
+        width: 25px;
+        top: calc(60% + 4px);
+      }
+    }
+  }
+}
+
+.navbar-menu_mobile{
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 2;
+  background-color: rgba(0, 0, 0, 0.85);
+  transition: 0.5s ease-out;
+  padding: 0;
+  &.is-opacity{
+    z-index: -1;
+  }
+  .menu-list{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 50%;
+    margin: 0 auto;
+    li:first-child {
+      margin: 0;
+    }
+  }
+  .menu-list__item{
+    &:hover,&:active,&:focus{
+      background-color: transparent;
+    }
+    a{
+      font-size: 20px;
+      color: #fff;
+      text-transform: uppercase;
+      text-align: center;
+      line-height: 25px;
+      &:hover,&:active,&:focus{
+        outline: none;
+        background-color: transparent;
+      }
+    }
+    button{
+      width: 100%;
+      &:hover,&:active,&:focus{
+        background-color: transparent;
+        outline: none;
+        box-shadow: none;
+        border-color: initial;
+      }
+    }
+  }
+}
+
+.navbar_fixed{
```

Comment:
Use just dash to be consistent with the rest of the codebase (eg. slick)

Reply:
fixed